Library/Homebrew/extend/ARGV.rb

```diff
@@ -156,15 +158,16 @@ def values(name)
   end
 
   def force?
-    flag? "--force"
+    flag?("--force") || Homebrew::CLI::Parser.args.force?
   end
 
   def verbose?
-    flag?("--verbose") || !ENV["VERBOSE"].nil? || !ENV["HOMEBREW_VERBOSE"].nil?
+    flag?("--verbose") || !ENV["VERBOSE"].nil? || !ENV["HOMEBREW_VERBOSE"].nil? ||
+      Homebrew::CLI::Parser.args.verbose?
```

Comment:
Instead of adding an `or` condition (`|| Homebrew::CLI::Parser.args.verbose?`) wherever `ARGV.verbose?` has been used throughout codebase, I'm adding it here and ideally would replace occurrences `ARGV.verbose?` with `Homebrew::CLI::Parser.args.verbose?` once we use `Parser` everywhere else.

Reply:
I think it would be good to not add any references to `Homebrew::CLI::Parser.args` or `Homebrew.args` in `ARGV` but instead work towards replacing calls to `ARGV` with calls to `Homebrew.args` instead.